.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
+max_line_length = 120
```

Comment:
I didn't find [any spec](https://editorconfig-specification.readthedocs.io/) of this rule. I'm wondering if you have any link which can describe it.

Reply:
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length

Reply:
I see. But it does not work well on IntelliJ IDEA.

![image](https://user-images.githubusercontent.com/1450685/130025120-315dc201-2836-4525-96e8-e9585099357e.png)

```
IntelliJ IDEA 2019.2.2 (Community Edition)
Build #IC-192.6603.28, built on September 6, 2019
Runtime version: 11.0.3+12-b304.56 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.7
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Registry: 
Non-Bundled Plugins: Native2Ascii, com.shuqi.tools.translate-properties, native2asciiplug, Docker
```

Reply:
See also [editorconfig: remove max_line_length directives #7761](https://github.com/MaterializeInc/materialize/pull/7761)

Reply:
Since this rule is going to be removed, let's remove it?

Reply:
+1 remove it